pkgs/top-level/all-packages.nix

```diff
@@ -6483,6 +6483,11 @@ with pkgs;
 
   rush = callPackage ../shells/rush { };
 
+  termonad = callPackage ../shells/termonad {
```

Comment:
This should probably not be placed in the section with other shells, but instead with other terminal emulators.  It should go near where `roxterm`, `sakura`, etc are defined.

Reply:
Also, I think this should maybe be called `termonad-with-packages` (to match the `xmonad-with-packages` derivation).

Reply:
I'll move it.
I figured if you wanted "termonad-without-packages", you could just grab it from "haskellPackages", so no need to suffix it.

Reply:
Yeah, I think `termonad` is the better name as well.

However, in this case I think it might be better to keep a similar naming scheme with other similar packages (`xmonad-with-packages` is the only one I know of), rather than using something different.

Reply:
I'll rename it and then I think we should be good to go